posthog/models/filters/mixins/funnel.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from posthog.constants import FUNNEL_STEP, FUNNEL_WINDOW_DAYS
+from posthog.models.filters.mixins.base import BaseParamMixin
+from posthog.models.filters.mixins.utils import cached_property, include_dict
+
+
+class FunnelWindowDaysMixin(BaseParamMixin):
+    @cached_property
+    def funnel_window_days(self) -> Optional[int]:
+        _days = int(self._data.get(FUNNEL_WINDOW_DAYS, "0"))
```

Comment:
Is there a specific reason behind removing the default window?

Reply:
oh, and while you're at it, remove the `funnel_window_days.py` file?

Reply:
Asking because this might be a breaking change for the API, assuming there's any call coming to us without the window size

Reply:
I want to get rid of magic numbers happening everywhere and not to have to take into account defaults. Also, if you put a default here, the filter dict will always contain this value so our cache keys will be really inconsistent because queries that don't need this parameter will be cached against it